src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -213,10 +206,9 @@ public void testDelete_GenericExceptionResponse() {
 
         when(restTemplate.exchange(eq(DELETE_URL), eq(HttpMethod.DELETE), any(), eq(String.class))).thenThrow(exception);
 
-        expectedException.expect(RestClientException.class);
-        expectedException.expectMessage(exception.getMessage());
-
-        fileTransferApiClient.delete(FILE_ID);
+        RestClientException thrown = assertThrows(RestClientException.class, () ->
+            fileTransferApiClient.delete(FILE_ID));
+        assertEquals(thrown.getMessage(), thrown.getMessage());
```

Comment:
same here:
assertEquals(thrown.getMessage(), thrown.getMessage()); will never be false, it's the same as saying does a == a
we need to check that the message in the thrown exception is what is expected
assertEquals(EXCEPTION_MESSAGE, thrown.getMessage());